riva/client/nmt.py

```diff
@@ -22,10 +22,16 @@ def streaming_s2t_request_generator(
     for chunk in audio_chunks:
         yield riva_nmt.StreamingTranslateSpeechToTextRequest(audio_content=chunk)
 
+def add_dnt_phrases_dict(req, dnt_phrases_dict):
+    dnt_phrases = [f"{key}##{value}" for key, value in dnt_phrases_dict.items()]
```

Comment:
dnt_phrases_dict can be None as well, if nothing is passed. In that case, it will cause error.
can we handle this case? see similar fix for custom dictionary https://github.com/nvidia-riva/python-clients/commit/6f7bd6bc76352df1b392173ba8339bda9cc4a87b

Reply:
this is still not correct, dnt_phrases_dict in this function can be None, so dnt_phrases_dict.items() will throw error. Need to add None check.

Reply:
I think I missed it.

Reply:
it looks fine now